recipes/flask-migrate/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{%set name = "Flask-Migrate" %}
+{%set version = "1.5.1" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "4af99e7d581f1169e774662f69fd913608126d8925614432dc2b42144bc328cb" %}
+{%set build_num = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  preserve_egg_dir: True
+  number: {{ build_num }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - flask >=0.9
+    - flask-sqlalchemy >=1.0
+    - alembic >=0.6
+    - flask-script >=0.6
```

Comment:
Appears we were missing some OS X packages for this dependency. Please see issue ( https://github.com/conda-forge/flask-script-feedstock/issues/1 ) for details.